### PR TITLE
jjb: fix package filter of submit-project

### PR DESCRIPTION
**Pull request by jdsn**
Support blacklist and whitelist filter for package names to allow for easier configuration of the jobs (no need to list all whitelisted packages, instead list the unwanted ones).

Revert (partially) the wrong commit cdca70f

Adapt package filter in cloud6 cd-gating job.

### CODE REVIEW

File: jenkins/ci.suse.de/cloud-mkcloud6-cd-gating.yaml [hunk 1]
```diff
@@ -38,4 +38,4 @@
           predefined-parameters: |
             project=Devel:Cloud:6:SAP
             subproject=Staging
-            packagefilter=crowbar python openstack patterns release-notes-suse-openstack-cloud suse-cloud-upgrade
+            package_blacklist=ardana
```
Comment (rsalevsky) [inline jenkins/ci.suse.de/cloud-mkcloud6-cd-gating.yaml]: There is no ardana on Cloud 6, so we can remove it.

**Comment by jdsn** [comment]
This supersedes PR #2442

**Comment by bmwiedemann** [comment]
1st commit message has typo: `seprated`
and the 2nd and 3rd commit message seem to be swapped

**Comment by jdsn** [comment]
@bmwiedemann why should commit 2 and 3 be swapped?

**Comment by dirkmueller** [comment]
fine for me, although I think implementing both white and blacklist feels like overkill.. everything is fine as it is right now, isn't it?

**Comment by jdsn** [comment]
@dirkmueller 
> I think implementing both white and blacklist feels like overkill

The blacklist is used in cloud-mkcloud-gating-template file, this makes it easier to catch all packages but the listed ones.
With the old filter you had to list all packages explicitly (which made the filter useless, until ardana came) and regularly maintain the list. I just didn't want to drop the whitelist filter (it was one before). ... and it was just too easy to have both.

**Comment by jdsn** [comment]
@dirkmueller 

> everything is fine as it is right now, isn't it?

Well, this PR (is replacing PR #2442 and) is needed to stop copypacking ardana packages to non-Staging. This was requested by you.

**Comment by jdsn** [comment]
> fine for me

I'm regarding this as approval :)

**Pull request merged**